docs/examples/import-definitions/rabbitmq.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: rabbitmq.com/v1beta1
+kind: RabbitmqCluster
+metadata:
+  name: import-definitions
+spec:
+  replicas: 1
+  override:
+    statefulSet:
+      spec:
+        template:
+          spec:
+            containers:
+            - name: rabbitmq
+              volumeMounts:
+              - mountPath: /tmp/definitions
```

Comment:
I think if we were to be suggesting an actual path it should be in `/var` as this file could be expected to grow.

The rabbitMQ docs opt for `/path/to/exported/definitions.json` which saves us having to bother about Linux conventions

Reply:
Sure